core/templates/pages/exploration-editor-page/modal-templates/save-version-mismatch-modal.component.spec.ts

```diff
@@ -57,9 +57,9 @@ class MockWindowRef {
         if (this._hashChange === null) {
           return;
         }
-        this._hashChange();
+        this._hashChange;
```

Comment:
Why is this changed?

Reply:
_hasChange is not a function, it's a variable. It was a mistake so I rectified this.